Dockerfile

```diff
@@ -49,8 +37,7 @@ RUN cp config/bioportal_config_env.rb.sample config/bioportal_config_production.
  && cp config/database.yml.sample config/database.yml
 
 RUN if [ "${RAILS_ENV}" != "development" ]; then \
-  bundle exec bootsnap precompile --gemfile app/ lib/ && \
-  SECRET_KEY_BASE_DUMMY=1 ./bin/rails assets:precompile; fi
```

Comment:
`SECRET_KEY_BASE_DUMMY=1 ./bin/rails assets:precompile;` is also required, otherwise the UI will not display correctly.